.gitignore

```diff
@@ -77,6 +76,7 @@ target/
 
 # Virtual environments
 .venv
+.python-version
```

Comment:
Why did you want this added? This is typically a local file used with pyenv to automatically activate the python virtualenv. This could be different on someone else's machine. This file is created when running `pyenv local <virtualenv_name>`.

Reply:
Yeah unfortunately this file is used for 2 different purposes by pyenv. I wanted to add it to track the python version (3.6.6) but it conflicts with pyenv virtualenv usage, so I had to gitignore it instead.